recursion/get_maze_path.js

```diff
@@ -0,0 +1,22 @@
+const get_maze_path = (sr, sc, dr, dc) => {
+    // Base case
+    if(sr > dr || sc > dc) return;
+
+    if( sr === dr && sc === dc) return [""];
+    // Recursion
+    const horizontal_paths = get_maze_path(sr, sc + 1, dr, dc);
+    const vertical_paths = get_maze_path(sr + 1, sc, dr, dc);
+
+    const paths = [];
+
+    for(i in horizontal_paths){
+        paths.push('h'+horizontal_paths[i]);
+    }
+
+    for(i in vertical_paths){
+        paths.push('v' + vertical_paths[i]);
+    }
```

Comment:
Same as above. 

Longer the code goes, harder it becomes to understand this single character variable names. (i, v)